client/db.go

```diff
@@ -0,0 +1,461 @@
+package client
+
+import (
+	"encoding"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/cockroachdb/cockroach/proto"
+	"github.com/cockroachdb/cockroach/testutils"
+	gogoproto "github.com/gogo/protobuf/proto"
+)
+
+// Timestamp ...
+type Timestamp struct {
+	WallTime int64
```

Comment:
Ok, I was wondering about that. Removal simplifies this down to an `int64` which is easily converted to a time.Time.